data_management/mysqldumpall/dumpall.sh

```diff
@@ -80,29 +81,29 @@ for database_name in $database_names; do
 
   # Get all table names from database file
   echo Get all table names from database file
-  tables=$(grep --text -P '^-- Table structure for table ' "$tempdb_dir"/"$database_name".sql | awk '{ print $NF }' | sed 's/`//g');
+  tables=$(grep --text -P '^-- Table structure for table ' "$tmp_dir"/"$database_name"-temp/"$database_name".sql | awk '{ print $NF }' | sed 's/`//g');
 
   for table in $tables; do
     # Extract table
     echo Extract table $database_name.$table
-    $mysqldumpsplitter_command --source "$tempdb_dir"/"$database_name".sql --extract TABLE --match_str "$table" --compression none --output_dir "$tempdb_dir"/tables-temp >&2
+    $mysqldumpsplitter_command --source "$tmp_dir"/"$database_name"-temp/"$database_name".sql --extract TABLE --match_str "$table" --compression none --output_dir "$tmp_dir"/"$database_name"-temp/tables-temp >&2
 
     # Remove comments starts with -- in table sql
     echo Remove comments starts with -- in table sql $database_name.$table
-    grep -vw --text '^--' "$tempdb_dir"/tables-temp/"$table".sql > "$target_dir"/"$database_name"/tables/"$table".sql
+    grep -vw --text '^--' "$tmp_dir"/"$database_name"-temp/tables-temp/"$table".sql > "$target_dir"/"$database_name"/tables/"$table".sql
   done
 
   # Remove comments starts with -- in the main database sql 
   echo Remove comments starts with -- in the main database sql $database_name
-  grep -vw --text '^--' "$tempdb_dir"/"$database_name".sql > "$tempdb_dir"/database.sql
+  grep -vw --text '^--' "$tmp_dir"/"$database_name"-temp/"$database_name".sql > "$tmp_dir"/"$database_name"-temp/database.sql
 
   # Copy only the selected line base on the starting text of the line
-  grep --text -wE '(^CREATE DATABASE|^USE)' "$tempdb_dir"/database.sql > "$target_dir"/"$database_name"/database.sql
+  grep --text -wE '(^CREATE DATABASE|^USE)' "$tmp_dir"/"$database_name"-temp/database.sql > "$target_dir"/"$database_name"/database.sql
 
   # Remove databases and tables with comments
   echo Remove databases and tables with comments $database_name
-  rm -r "$tempdb_dir"
+  rm -r "$tmp_dir"/"$database_name"-temp
 done
 
-# Remove temp file of main data
-rm "$single_dump_file"
+# Remove temp folder
+rm "$tmp_dir"
```

Comment:
This looks like it will fail because `$tmp_dir` is not empty. Can you double-check that the script actually completes correctly? (Or maybe you've tested it already, just let me know.)

Reply:
Yup. I forgot to add -r. I just updated it on the latest commit.